tests/foreman/ui/test_computeresource.py

```diff
@@ -128,18 +131,23 @@ def test_create_libvirt_resource_different_names_negative(self, name):
 
         """
         with Session(self.browser) as session:
-            make_resource(
-                session,
-                name=name,
-                provider_type=FOREMAN_PROVIDERS['libvirt'],
-                parameter_list=[['URL', self.current_libvirt_url, 'field']],
-            )
-            self.assertIsNotNone(self.compute_resource.wait_until_element(
-                common_locators["name_haserror"]
-            ))
-
-    @data(*invalid_names_list())
-    def test_create_libvirt_resource_description_negative(self, description):
+            for name in invalid_values_list():
```

Comment:
That change will work improperly in terms of logic (as tab will not be inputted into field), but impact will be minimal, so I think we can leave as it is
I use `' ', *invalid_names_list()` instead `invalid_values_list()` intentionally and so you should be accurate in UI cases when you perform fixes like this